crates/bevy_pbr/src/render/mesh_view_bindings.rs

```diff
@@ -144,190 +148,142 @@ impl From<Option<&ViewPrepassTextures>> for MeshPipelineViewLayoutKey {
     }
 }
 
+fn buffer_layout(
```

Comment:
this could be flattened out too, if `render_device::get_supported_read_only_binding_type` returned a builder

Reply:
I tried doing it but I'm not seeing how I'd do that. It doesn't really work because I'm not sure how I'd set the min_binding_size.

Reply:
right, it's not as straightforward as i thought.

the better refactor is probably to make GpuPointLights and ViewClusterBindings use GpuArrayBuffer, instead of having their own Storage/Uniform enum internally. future work.